doc/doxygen/udffsck.txt

```diff
@@ -0,0 +1,101 @@
+
+/** 
+\page udffsck-doc udffsck documentation
+\tableofcontents
+Author: **Vojtech Vladyka** <<vojtech.vladyka@gmail.com>>
+
+Developer documentation for **udffsck** tool.
+
+\section purpose Purpose
+Purpose of this tool is check and correction of UDF filesystem metadata. UDF standard doesn't cover data integrity itself, it is covered by used medium (most usually ECC blocks).
+
+In case of found errors on filesystem, this tool can try to fix them. Fixing process is irreversible and writes changes back to checked medium. You should always back up your data before.
+
+\section limitations Limitations
+There are several limits for this tool. 
+ - it can work with UDF up to version 2.01
+ - it can't fix WORM devices (obviously)
```

Comment:
It is not /obvious/. UDF also for WORM media (with VAT) is designed to recover from errors (e.g. by incrementelly writing new data correctly and then writing new correct VAT table).

So it is just a limitation of current implementation.